### ISSUE TITLE
Consistently resolving arg structure alternatives

### ISSUE THREAD

**Issue by ejwood19** [isi-nlp/DiplomacyAMR#3]
_TL;DR where there are two arg structure options for AMR and we expect sentence meaning to be equivalent in our data, is there a default principle that would work, such as taking the option that distinguishes fewer ARGs in the top frame (e.g. for PREVENT and ALLY below)? Or since we have a limited number of frames, should we just resolve frame by frame?_

How to get consistent annotation for frames which allow multiple arg structure options that don't make a significant difference for Diplomacy data?
For example:

**1. PREVENT**
(1) X prevented Y’s attack on Z
(2) X prevented Y from attacking Z
(3) X prevented the attack on Z     [where Y is known to be the attacker from a previous sentence]

prevent-01 has
* ARG1: Theme (action or object being prevented)   theme
* ARG2: secondary predication or action   theme

So assuming these are all to have the same AMR, do we want to default to 
p / prevent-01
     :ARG0 X
     :ARG1 Y
     :ARG2 attack (:ARG0 Y, :ARG1 Z)

or

 p / prevent-01
     :ARG0 X
     :ARG1 attack (:ARG0 Y, :ARG1 Z)

**2. ALLY**
ally-01 has the option to distinguish 
ARG0: agent, causer  
ARG1: entity united
ARG2: entity united with ( co-agent)
(4) X is forming an alliance with Y
(5) X and Y are forming an alliance

a / ally-01
     :ARG1 X
     :ARG2 Y

a / ally-01
     :ARG1 a2 / and  (:op1 X, :op2 Y)

PROPOSAL: would it work to pick the option that has fewer distinct ARGs and use it consistently? i.e. take the second option above for both _prevent_ and _ally_ ? This would be consistent with what we've done so far for "move" and "retreat", plus the result-oriented annotation for "build".

ALTERNATIVE: just have a frame list where we pick one by one the version that seems most appropriate for our purposes.
